driver-core/src/main/java/com/datastax/driver/core/RegularStatement.java

```diff
@@ -189,21 +189,4 @@ public boolean hasValues() {
         return hasValues(CodecRegistry.DEFAULT_INSTANCE);
     }
 
-    /**
-     * Returns this statement as a CQL query string.
```

Comment:
This is not true anymore as this method now prints much more than just the query string. Ironically, these very same javadocs state a few lines below that the query string only "does not convey all the information stored in `Statement` objects".